modules/appeals_api/app/controllers/appeals_api/supplemental_claims/v0/supplemental_claims_controller.rb

```diff
@@ -23,7 +23,7 @@ def index
       veteran_scs = AppealsApi::SupplementalClaim.select(ALLOWED_COLUMNS)
                                                  .where(veteran_icn: params[:icn])
                                                  .order(created_at: :desc)
-      render json: AppealsApi::SupplementalClaimSerializer.new(veteran_scs).serializable_hash
+      render_supplemental_claim(veteran_scs)
```

Comment:
(I realized this syntax was equivalent, so I reused the `render_supplemental_claim` method)